asf_levies_model/summary.py

```diff
@@ -0,0 +1,227 @@
+import pandas as pd
+from typing import Dict, Optional
+
+
+def _sum_levies(val: float, summary: str, fuel: str, levies: list) -> float:
+    """Calculate sum of levies."""
```

Comment:
I think it's worth documenting that the "summary" parameter for this function can only be "fixed" or "variable, and not "total" (or perhaps can be renamed?).
I know at present that only "fixed" or "variable" is passed to it in _calculate_policy_costs() but just in case it will be used in a new function later on.

```suggestion
    """Calculate sum of levies.

    Parameters
    ----------
    val : float
        Gas or electricity consumption value.
    summary : str
        Charging basis, can be 'fixed' or 'variable'.
    fuel : str
        Fuel type, can be 'gas' or 'electricity'.
    levies : list
        Collection of levies used to estimate policy costs.
    Returns
    -------
    float
        Policy cost component value for charging basis and fuel type given.
    """
```

Reply:
Originally I didn't give this a full docstring because it's an internal function. However, I think you're right that we're likely to want to reuse it as the model evolves, so this provided docstring is much appreciated!